features/git-town/configuration/non_feature_branches/modify_non_feature_branch_configuration.feature

```diff
@@ -40,19 +58,30 @@ Feature: modifying the non-feature branch configuration
   Scenario: removing a non-feature branch that exists
     Given my non-feature branches are "staging" and "qa"
     When I run `git town non-feature-branches --remove staging`
-    Then I see "Removed 'staging' from non-feature branches"
+    Then I see no output
     And the non-feature branches don't include "staging"
     And the non-feature branches include "qa"
 
 
   Scenario: using invalid parameter
     Given my non-feature branches are "staging" and "qa"
-    When I run `git town non-feature-branches --invalid-parameter staging`
-    Then I see "usage: git town non-feature-branches (--add | --remove) <branchname>"
+    When I run `git town non-feature-branches --invalid-parameter staging` while allowing errors
+    Then I see
+      """
+
+        Error
+        unknown option: --invalid-parameter
+
+      """
 
 
   Scenario: missing branch name
     Given my non-feature branches are "staging" and "qa"
-    When I run `git town non-feature-branches --add`
-    Then I see "Missing branch name"
-    And I see "usage: git town non-feature-branches (--add | --remove) <branchname>"
+    When I run `git town non-feature-branches --add` while allowing errors
+    Then I see
+      """
+
+        Error
+        branch name required
```

Comment:
Customer perspective: "Don't just tell me that what I did is wrong. Either fix it, or help me how to do it better."

If the user forgot to provide the branch name (as the error implies), I would assume that they don't know the syntax. In general it is better to err on the side of being helpful than too logical and assuming, i.e. show how to do this right here. 

Maybe the user didn't remember the correct syntax and just typed something. I think in this situation (and in almost all others where the user enters something incorrectly) it would be nice and helpful to show the correct syntax.